### PR TITLE
Add LPF tree sitter

### PR DESCRIPTION
**Pull request by TheZoq2**
Adds a small tree-sitter parser for the LPF configuration language https://github.com/danderson/ulxs/blob/main/lpf.md

### CODE REVIEW

File: runtime/queries/lpf/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "SYSCONFIG"
+  "BLOCK"
+  "LOCATE"
+  "COMP"
+  "FREQUENCY"
+  "PORT"
+  "IOBUF"
+] @keyword
+
+["SITE"] @storage
```
Comment (the-mikedavis) [inline runtime/queries/lpf/highlights.scm:11]:
```suggestion
["SITE"] @keyword.storage
```

to match https://docs.helix-editor.com/master/themes.html#syntax-highlighting

**Review by the-mikedavis — COMMENTED** [review]
You can run `cargo xtask docgen` and commit the results to fix the docs CI

**Review by the-mikedavis — CHANGES_REQUESTED** [review]
Actually, looks like this still needs the change from `cargo xtask docgen`

**Comment by TheZoq2** [comment]
Yeah, sorry, I forgot to make that change. Give me a minute

**Comment by TheZoq2** [comment]
Fixed

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**